tests/policy_snapshot/snapshots/AWS-CRT-SDK-TLSv1.3-2025-PQ-KX-Required

```diff
@@ -0,0 +1,32 @@
+name: AWS-CRT-SDK-TLSv1.3-2025-PQ-KX-Required
+min version: TLS1.3
+rules:
+- Perfect Forward Secrecy: yes
+- FIPS 140-3 (2019): no
+cipher suites:
+- TLS_AES_128_GCM_SHA256
```

Comment:
Should we drop this? My understanding is that AES-128 isn't considered acceptable for PQ.

The existing default_pq policy has the same "bug" but there it's at least also accepting non-pq fallback...

Reply:
We still consider AES-128 to be PQ-secure. AES-128 has equivalent security to NIST's Level 1 PQ algorithms (ML-KEM-512 for example).